apiserver/jupyter_notebook_config.py.in

```diff
@@ -0,0 +1,767 @@
+# Configuration file for jupyter-notebook.
```

Comment:
I'd prefer to strip all the comments, otherwise the active code gets lost in a sea of noise.